### ISSUE TITLE
Output test coverage

### ISSUE THREAD

**Issue by namoscato** [cedpoilly/vue-gapi#57]
Originally referenced in some Karma configuration; this would be nice to wire up at some point.

**Comment by cedpoilly** [comment]
Yes, let's keep it for later though. We can implement that once we have a stable testing setup 😉
